spec/griddler/cloudmailin/adapter_spec.rb

```diff
@@ -20,6 +41,17 @@
     })
   end
 
+  it 'normalizes parameters where the recipient has been BCCed' do
+    expect(Griddler::Cloudmailin::Adapter.normalize_params(bcc_params)).to be_normalized_to({
+      to: ['Some Identifier <some-identifier@example.com>'],
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.